check/src/main/scala/rsc/diffoutline/Main.scala

```diff
@@ -2,13 +2,39 @@
 // Licensed under the Apache License, Version 2.0 (see LICENSE.md).
 package rsc.diffoutline
 
+import cats.implicits._
+import com.monovore.decline.Opts.{flag, option}
+import com.monovore.decline.{CommandApp, Opts}
 import java.nio.file._
 import rsc.checkbase._
 import rsc.checkoutline
+import scala.reflect.ClassTag
 
-object Main extends MainBase[Settings, Int, Path, Path] {
-  def settings(args: List[String]) = {
-    Settings.parse(args)
+object Main extends CommandApp(MainCommand.command)
+
+private[rsc] object MainCommand extends MainBase[Settings, Int, Path, Path] {
```

Comment:
This is pretty difference in usage vs `diffoutline` from before, which had
`val usage = "diffoutline [--quiet] <nsc_classpath> <rsc_classpath>"`

Reply:
I misunderstood the initial usage.  Will fix.